### DISCUSSION TITLE
moving all indexes off search node

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#13419] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

How do you push all the indexes off a search node before removing it.  I modified them to only data_cold. 
Thank you for all your help. 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
First, do you have another search node that can take cold indices? If so, you could follow this https://docs.securityonion.net/en/2.4/removing-a-node.html#removing-a-search-node Just change the IP in the command to your search node's IP.
